Automattic/jetpack
# VideoPress: TypeScript video block

This PR re-writes the VideoPress video block with TS. Thanks to it, it also fixes a bug when setting the block `src` attribute.

Fixes #

## Proposed changes:

* VideoPress: TypeScript video block. Fix setting src block attribute.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Restart the building process. (Stop the current ones)
* Run build
```
jetpack build packages/videopress
```
*  Run build&watch
```
jetpack watch packages/videopress
```
* Confirm the app builds the source properly
* Confirm the block works as expected: hard-refresh. Add, edit, etc.
